tests/test_utils.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+
+from maskit.utils import cross_entropy
+
+
+def test_cross_entropy():
+    predictions = np.array([[0.25, 0.25, 0.25, 0.25], [0.01, 0.01, 0.01, 0.96]])
```

Comment:
The test case should be 

`predictions = np.array([[0.25, 0.25, 0.25, 0.25], [0.01, 0.01, 0.01, 0.97]])`

So the predictions add up to one. Then it is also identical to the output of the sklearn.metrics implementation: `0.7085330098933221` and not `0.71355817782`

Reply:
Hm, interesting comment.
This might actually change our use case as in the quantum case we will usually not add up to `1.0`. What is the expectation you have? Should we get the same result as with sklearn or not? So should we introduce a normalisation to always add up to the sum of target probabilities?

Reply:
Should we always add up to `1` in the quantum case as long as we have consider all of the `2^n` measured outputs? For the iris dataset with three classes, I would prefer if we would perform a softmax before calling the cross entropy.

Reply:
Not softmax, sorry. That doesn't make any sense. Adding a normalisation should be our best option.